addons/vikhr/CfgMagazines.hpp

```diff
@@ -0,0 +1,53 @@
+class CfgMagazines {
+    class 2Rnd_LG_scalpel;
+    class 6Rnd_LG_scalpel;
+    class 8Rnd_LG_scalpel;
+    class PylonRack_1Rnd_LG_scalpel;
+    class PylonMissile_1Rnd_LG_scalpel;
+    class PylonRack_3Rnd_LG_scalpel;
+    class PylonRack_4Rnd_LG_scalpel;
+
+    class GVAR(2Rnd): 2Rnd_LG_scalpel {
+        author = "Dani (TCVM)";
+        displayName = CSTRING(2x);
+        ammo = QGVAR(9k121);
```

Comment:
Since these all inherit uniquely from base classes, I can't reasonably do this without copying config data from Arma, which would be counter intuitive to the request